docs/static/breaking-changes.asciidoc

```diff
@@ -1,5 +1,14 @@
 [[breaking-changes]]
-== Breaking Changes
+
+== Breaking Changes Across PQ Versions
+
+If you are upgrading Logstash and use the persistent queue, we strongly recommend that you drain or delete the persistent queue before you upgrade.
+
+To drain the queue, enable the `queue.drain` setting, and then shutdown Logstash. Wait for it to shutdown completely. This may take a while if you have a large queue backlog.
+
+We are working to resolve issues with data incompatibilities in our 6.3.0 release. These steps won’t be required when you upgrade to versions greater than or equal to 6.3.0.
```

Comment:
This phrase sounds a bit like you wont need to drain the queue if you are going from 6.2.0 to say 6.4.0, but that's not correct, right? I would rather explain that we increase the major version in 6.3.0 and any upgrade from a version < 6.3.0 to a version >= 6.3.0 should start with draining the queue.

Reply:
That's a good point. I'll clarify that.